Packages/Settings/Sources/SettingsUI/Internal/GitHubSettings/GitHubSettingsView.swift

```diff
@@ -14,6 +14,21 @@ struct GitHubSettingsView: View {
     var body: some View {
         Form {
             Section {
+                Picker("Version", selection: $viewModel.version) {
+                    ForEach(GitHubServiceVersion.Kind.allCases, id: \.self) { scope in
+                        Text(scope.title)
+                    }
+                }
```

Comment:
Would it make sense to turn this picker into a segmented control to align with the Runner Scope setting?

```suggestion
                }
                .pickerStyle(.segmented)
```